docs/tempo/website/getting-started/traces.md

```diff
@@ -0,0 +1,69 @@
+
+Finally, the request and response activity ends and a record of that request is sent to Grafana or Grafana Cloud.
+
+## Traces and labels
+
+Setting up tracing adds a label to all of these events. The label is generated when the request is initiated and that same label is applied to every single event as the request and response generate activity across the system.
+
+That label enables one to trace, or follow a request as it flows from node to node, service to microservice to lambda function to wherever it goes in your chaotic, cloud computing system and back again. This is recorded and displayed as spans.
+
+Here's an example showing two pages in Grafana Cloud. The first, on the left (1), shows a query using the **Explore** feature. In the query results you can see a **traceID** field that was added to an application. That field contains a **Tempo** label. The second page, on the right (2), uses the same Explore feature to perform a Tempo search using that **traceID** label. It then shows a set of spans as horizontal bars, each bar denoting a different part of the system.
+
+<p align="center"><img src="../assets/trace-explore-spans.png" alt="Trace example"></p>
+
+The label is applied to activities recorded as metrics and as logs. 
+
+## What are traces used for? 
+
+Traces can help you find bottlenecks. A trace can be visualized to give a graphic representation of how long it takes for each step in the data flow pathway to complete. It can show where new requests are initiated and end, and how your system responds. This data helps you locate problem areas, often in places you never would have anticipated or found without this ability to trace the request flow.
+
+Metrics, logs, and traces form the three pillars of observability. Metrics provide the health data about the state of a system. Logs provide an audit trail of activity that create an informational context. Traces tell you what happens at each step or action in a data pathway. 
+
+## Terminology
+
+Data source
+: A basic storage for data such as a database, a flat file, or even live references or measurements from a device. dA file, database, or service that provides data. For example, traces data is imported into Grafana by configuring and enabling a Tempo data source.
+
+Exemplar
+: Any data that serves as a detailed example of one of the observations aggregated into a metric. An exemplar contains the observed value together with an optional timestamp and arbitrary labels, which are typically used to reference a trace.
+
+Log
+: Audit trail of activity used to diagnose problems. Logs can provide informational context, such as detailed records of all events during user interactions, for example, when events happen, who used the system, status messages, etc.
+
+Metric
+: A number, helping operators understand the state of a system, for example, number of active users, error count, average response time, etc. 
+
+Span
+: Represents a unit of work done within a trace.
```

Comment:
```suggestion
: Represents a unit of work done within a trace.
```
Why "Represents" and not simply  "A unit of work done within a trace."

Reply:
Updated.